### DISCUSSION TITLE
请教一下有rapidTable的微调指印吗

### DISCUSSION THREAD

**Discussion by enymfc25173** [RapidAI/RapidTable#24] [Q&A] [answered]
请教一下有rapidTable的微调指印吗

**Comment by SWHL** [comment, marked as answer]
RapidTable仅是推理库，如果想要微调，需要移步到PP-Strurcture中了。相关文档：https://paddlepaddle.github.io/PaddleOCR/latest/ppstructure/model_train/train_table.html
